package.json

```diff
@@ -27,7 +27,8 @@
   "license": "MIT",
   "dependencies": {
     "axios": "^0.19.2",
-    "crypto-js": "^4.0.0"
+    "crypto-js": "^4.0.0",
+    "data.js": "^0.12.11"
```

Comment:
Looks like we're using this only for the tests so maybe it can be a dev dependency?

Reply:
I need to check with @rufuspollock  if he wants to use "data.js" to do something into the SDK. Is not clear for me atm